index.html

```diff
@@ -604,6 +616,149 @@ <h3>Set the session description</h3>
+        attribute EventHandler oncandidatepairadded;
+        attribute EventHandler oncandidatepairpruneproposed;
+      };</pre>
+    <section>
+      <h2>Attributes</h2>
+      <dl data-link-for="RTCIceTransport" data-dfn-for="RTCIceTransport" class="attributes">
+        <dt>
+          <dfn>oncandidatepairadded</dfn> of type <span class="idlAttrType">{{EventHandler}}</span>
+        </dt>
+        <dd>
+          <p>
+            The event type of this event handler is {{candidatepairadded}}.
+          </p>
+          <p>
+            When the [= ICE agent =] has formed a candidate pair, the [= user agent =] MUST queue a task to [= fire an
+            event =] named {{candidatepairadded}} using the {{RTCIceCandidatePairEvent}} interface with the
+            {{RTCIceCandidatePairEvent/candidatePair}} attribute set to the formed candidate pair.
+          </p>
+        </dd>
+        <dt>
+          <dfn>oncandidatepairpruneproposed</dfn> of type <span class="idlAttrType">{{EventHandler}}</span>
+        </dt>
+        <dd>
+          <p>
+            The event type of this event handler is {{candidatepairpruneproposed}}.
+          </p>
+          <p>
+            When the [= ICE agent =] has selected a candidate pair to prune, but before the pruning has actually occurred,
+            the [= user agent =] MUST [= propose a candidate pair pruning =].
+          </p>
+        </dd>
+      </dl>
+    </section>
+    <section>
+      <h2>
+        <dfn>RTCIceCandidatePairEvent</dfn>
+      </h2>
+      <p>
+        The {{RTCIceTransport/candidatepairadded}} and {{RTCIceTransport/candidatepairpruneproposed}} events use the {{RTCIceCandidatePairEvent}} interface.
+      </p>
+      <div>
+        <pre class="idl">[Exposed=Window]
+interface RTCIceCandidatePairEvent : Event {
+  constructor(DOMString type, RTCIceCandidatePairEventInit eventInitDict);
+  readonly attribute RTCIceCandidatePair candidatePair;
+};</pre>
+        <section>
+          <h4>Constructors</h4>
+          <dl data-link-for="RTCIceCandidatePairEvent" data-dfn-for="RTCIceCandidatePairEvent" class="constructors">
+            <dt><dfn>RTCIceCandidatePairEvent.constructor()</dfn></dt>
+            <dd></dd>
+          </dl>
+        </section>
+        <section>
+          <h4>Attributes</h4>
+          <dl data-link-for="RTCIceCandidatePairEvent" data-dfn-for="RTCIceCandidatePairEvent" class="attributes">
+            <dt>
+              <dfn>candidatePair</dfn> of type <span class="idlAttrType">{{RTCIceCandidatePair}}</span>, readonly
+            </dt>
+            <dd>
+              <p>
+                The {{candidatePair}} attribute represents the {{RTCIceCandidatePair}} object associated with the event.
+              </p>
+            </dd>
+          </dl>
+        </section>
+      </div>
+      <div>
+        <pre class="idl">
+dictionary RTCIceCandidatePairEventInit : EventInit {
+  required RTCIceCandidatePair candidatePair;
```

Comment:
Done, unless we do add an interface for `RTCIceCandidatePair`.

I've kept `RTCIceCandidate` in the Init rather than `RTCIceCandidateInit`, consistent with [`RTCPeerConnectionIceEventInit`](https://www.w3.org/TR/webrtc/#dom-rtcpeerconnectioniceeventinit).